transports/quic/src/transport.rs

```diff
@@ -402,6 +328,7 @@ impl<P: Provider> Listener<P> {
         if self.is_closed {
             return;
         }
+        self.endpoint.close(From::from(0u32), &[]);
```

Comment:
Is this a correct error_code?

Reply:
AFAIK, we don't specify any error codes and these are passed verbatim to the other party. What did we set before?

Related: https://github.com/libp2p/specs/issues/479

Reply:
I think specifying `0` for now should be okay.

Reply:
Yes, we previously set `0` so this is fine.